### ISSUE TITLE
mupdf stopped shipping all Droid font variants

### ISSUE THREAD

**Issue by dpavlin** [koreader/kindlepdfviewer#230]
This makes build fail on clean machine. 

We at least have to have

ln -s /usr/share/fonts/truetype/droid/DroidSans-Bold.ttf fonts/droid/DroidSansBold.ttf

**Comment by houqp** [comment]
Should we ship Droid font by ourselves? I guess this command does not work in Mac.

**Comment by tigran123** [comment]
I was going to ask: why do we need to bind /usr/java/lib/fonts to ./fonts/hosts in kpdf.sh? I mean, is this just to add to the font selection available to the user or are any of those fonts actually used internally in KPV?

As far as I can see from grepping *.lua sources, we only use DroidSand and DroidSansMono font faces. Is this not correct? So, in principle we could ship just these two fonts and KPV would still work, right? If so, why do we need Dingbats? Surely it shouldn't be selected by the user (e.g. in crengine) for the main text?

**Comment by houqp** [comment]
Yes binding `/usr/java/lib/fonts` is just for extending font selection. And you are right, `Dingbats` should not be included...

**Comment by houqp** [comment]
I think it won't fail on clean build anymore, can we close it now?
